src/cudadecoder/Makefile

```diff
@@ -4,6 +4,7 @@ EXTRA_CXXFLAGS = -Wno-sign-compare
 include ../kaldi.mk
 
 ifeq ($(CUDA), true)
+ifeq ($(WITH_CUDAFEAT), 1)
```

Comment:
It might be better to do `ifneq ($(WITH_CUDAFEAT), 0)` since that would behave like the current Makefile behaves for already-configured setups.